src/state/shellscripts/bwrap.sh

```diff
@@ -93,6 +100,7 @@ case "$COMMAND" in
         add_mounts rw "$OPAM_SWITCH_PREFIX"
         add_mounts ro "$OPAM_SWITCH_PREFIX/.opam-switch"
         add_mounts rw "$PWD"
+        add_dune_cache_mount
```

Comment:
is it needed for install ? I don't think but not sure

Reply:
In principle, the "install" command is not even supposed to write to the source directory (this is a good rule even for `make`). It's not always true, and many packages write e.g. log files though, so we keep it rw.

I think in this case it's good hygiene to keep the mount only for the build. Anything breaking would be suspicious, and we can investigate and decide to be less strict at that point. (Anyway, packages don't use `dune` with the `install:` field usually, since that's done through `.install` files !)